wfe/web-front-end.go

```diff
@@ -380,24 +345,21 @@ func (wfe *WebFrontEndImpl) verifyPOST(logEvent *requestEvent, request *http.Req
 	// *anyway*, so it could always lie about what key was used by faking
 	// the signature itself.
 	if len(parsedJws.Signatures) > 1 {
-		err = core.SignatureValidationError("Too many signatures in POST body")
 		wfe.stats.Inc("WFE.Errors.TooManyJWSSignaturesInPOST", 1, 1.0)
 		logEvent.AddError("too many signatures in POST body: %d", len(parsedJws.Signatures))
-		return nil, nil, reg, err
+		return nil, nil, reg, probs.Malformed("Too many signatures in POST body")
```

Comment:
Okay! It's because originally all the things that called verifyPost also called sendError with its results and passed the text `"Unable to read/verify body"` as the `msg` parameter.

Now, we just use what ProblemDetails says. I fixed a some tests by putting that string back in at those places. I made that choice thinking it would make me consistent in all the places we cared about but perhaps that's not the case.